devices/stm32f107.yaml

```diff
@@ -61,6 +61,7 @@ _include:
  - ../peripherals/pwr/pwr_v1.yaml
  - ../peripherals/gpio/gpio_f1.yaml
  - ../peripherals/spi/spi_v1_without_FRF_FRE.yaml
+ - common_patches/tim/tim13_missing_opm.yaml
```

Comment:
```suggestion
```

In RM0009, TIM10/11/13/14 don't have OPM.

Reply:
RM0008 Rev 21 now has OPM bits